services/DataService.ts

```diff
@@ -70,16 +74,542 @@ const markers = new Map<PointOfInterest | string, MapMarker>([
   [
     '1',
     {
-      title: 'Boogschieten',
-      description: 'Hunger Games vibes',
+      title: 'Hoogteparcours EN zandkastelen en knikkerbanen',
+      description:
+        'Ouder dan 12 jaar maar toch nog zin om zandkastelen te bouwen en knikkerraces te houden? Op Saamdagen kan het!',
       latLng: {
-        latitude: 51.202028,
-        longitude: 4.848857,
+        latitude: 51.199612,
+        longitude: 4.851292,
       },
       icon: require('../assets/images/map-icons/icon-1.png'),
       layer: 'activities',
     },
   ],
+  [
+    '2',
+    {
+      title: 'Techniekenspel',
+      description:
+        'Ouder dan 12 jaar maar toch nog zin om zandkastelen te bouwen en knikkerraces te houden? Op Saamdagen kan het!',
```

Comment:
descriptions still missing? (same below)

Reply:
Yeah, didn't have time yet to look them up